src/main/io/gps.c

```diff
@@ -201,6 +220,20 @@ typedef struct ubxCfgRate_s {
     uint16_t timeRef;
 } ubxCfgRate_t;
 
+typedef struct ubxCfgValSet_s {
+    uint8_t version;
+    uint8_t layer;
+    uint8_t reserved[2];
+    uint8_t cfgData[MAX_VALSET_SIZE];
+} ubxCfgValSet_t;
+
+typedef struct ubxCfgValGet_s {
+    uint8_t version;
+    uint8_t layer;
+    uint16_t position;
+    uint8_t cfgData[MAX_VALSET_SIZE];
+} ubxCfgValGet_t;
+
```

Comment:
You can use polymorphism here because `ubxCfgValGet_t` and `ubxCfgValSet_t` are structurally exactly the same
```suggestion
typedef union ubxCfgValGetSet_u {
    ubxCfgValGet_t get;    
    ubxCfgValSet_t set;
} ubxCfgValGetSet_t;

```